test/basic-querying.test.js

```diff
@@ -958,6 +958,26 @@ describe('basic-querying', function() {
         null, // databases representing `undefined` as `null` (e.g. SQL)
       ]);
     });
+
+    describe('check __parent relationship in embedded models', () => {
+      it('should fill the parent in embedded model', async () => {
+        console.log();
```

Comment:
Ooops forgot an empty console.log line here for some reason...! Will remove that